src/BitcoinLib/BitcoinLib.csproj

```diff
@@ -7,6 +7,10 @@
     <Copyright>George Kimionis 2014 - 2017</Copyright>
     <Version>1.4.2.0</Version>
     <FileVersion>1.4.2.0</FileVersion>
+    <TargetFramework>netstandard2.0</TargetFramework>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
```

Comment:
With Paket and FAKE, is this reference needed? After all, without this reference, `./build.sh` still succeeds.

Reply:
You need this explicitly as users who import BitcoinLib as a submodule would face build errors without declaring it like that

Reply:
Gotcha. Then, it'd be awesome to include the submodule functionality on the README so users know about this?

Reply:
@GeorgeKimionis Yep, for Linux and Mac users, we'll also have to let them know the Mono runtime is required.